content/tutorials/spatial_crs_coding/index.Rmd

```diff
@@ -31,46 +30,64 @@ options(stringsAsFactors = FALSE)
 
 We will not go deeper into these components, because we want to focus on implementation.
 However it is highly recommended to read further about this, in order to better understand what a CRS means.
 A good contemporary resource in an R context is the section '_Coordinate reference systems: background_' in @bivand_ecs530_2019 (there is also an accompanying [video](https://www.youtube.com/watch?v=Rgn4Ns2UgAg&list=PLXUoTpMa_9s10NVk4dBQljNOaOXAOhcE0&index=4&t=0s) of that lesson).
 
-There are a few coordinated lists of CRSes around the globe, the most famous one being the [EPSG dataset](https://www.epsg.org), where each CRS has a unique _EPSG code_.
-You can consult these CRSes interactively at <http://www.epsg-registry.org> (official source) and through third-party websites such as <http://epsg.io>.
+There are a few coordinated lists of CRSs around the globe, the most famous one being the [EPSG dataset](https://www.epsg.org), where each CRS has a unique _EPSG code_.
+You can consult these CRSs interactively at <https://epsg.org> (official source) and through third-party websites such as <http://epsg.io>.
 For example, the 'World Geodetic System 1984' (WGS84) is a geodetic CRS with EPSG code `4326`, and 'Belge 1972 / Belgian Lambert 72' is a projected CRS with EPSG code `31370`.
 
-### How _was_ a CRS represented in R? Evolutions in PROJ and GDAL.
+### How _did_ we represent a CRS in R? Evolutions in PROJ and GDAL.
 
 It is good to know this, but you _can_ skip this section if you like.
 
 The reason for writing this tutorial are the recent (and ongoing) changes in several important geospatial libraries, especially **GDAL** and **PROJ**.
-They are used by most geospatial tools, including the key geospatial R packages `rgdal`, `sp`, `sf` and `raster`.
+They are used by most geospatial tools, including the key geospatial R packages `rgdal`, `sp`, `sf`, `stars`, `terra` and `raster`.
 
 Since long, coordinate reference systems in R (and many other tools) have been represented by so called 'PROJ.4 strings' (or 'proj4strings'), referring to the long-standing version 4 of the PROJ library.
-But, we will not use them here - it is counteradvised to use them any longer!
+But, we will not use them here!
+It is discouraged to use 'PROJ strings'[^projstring] any longer to represent a CRS; several string elements for CRSs are now deprecated or unsupported.
+Currently, PROJ (<https://proj.org>) regards PROJ strings only as a means of specifying a _coordinate operation_ (conversions or transformations _between_ CRSs).
+Performing coordinate operations is the main aim of PROJ.
 
-Let's just have **one last nostalgic peek** (and then, no more!!): the proj4string for the Belgian Lambert 72 CRS (EPSG:31370) "_was_":
+[^projstring]: 'PROJ string' is the term used in current PROJ documentation.
+Here, we only use 'PROJ.4 string' (or 'proj4string') when referring specifically to the PROJ string appearance in version 4 of the PROJ software.
+
```

Comment:
Maybe delete these lines (repetition of line 59)?

Reply:
This is in the footnote of the word 'PROJ string', explaining more explicitly why one time we speak about PROJ string and another time about PROJ.4 string. So I'd rather keep it, otherwise it must be guessed.